packages/website/bin/page-generator/get-package-info.js

```diff
@@ -117,6 +117,12 @@ module.exports = function getPackagesInfo(packagesPatterns, options = {}) {
       if (!fs.existsSync(readmePath)) {
         readmePath = '';
       }
+
+      let changelogPath = path.resolve(pkgPath, 'CHANGELOG.md');
+      if (!fs.existsSync(changelogPath)) {
+        changelogPath = '';
```

Comment:
Is an empty string actually appropriate? If there's no changelog it would be better to not have that property included in the output

Reply:
The path needs to be a string. A TypeError occurs if it is null or undefined

Reply:
So what's the behaviour when this empty string is used?

Reply:
The path will evaluate as false when determining which template string to use (i.e. with the component import or without)

Reply:
I think we should be more explicit in the code that this is what's happening. If we don't have a changelog then we should just immediately go to rendering the template sans component import instead of going through this flow.